ebextensions/waf.config

```diff
@@ -0,0 +1,55 @@
+Resources:
+  WafAcl:
+    Type: 'AWS::WAFv2::WebACL'
+    Properties:
+      Name: WafAcl
+      Scope: REGIONAL
+      DefaultAction:
+        Allow: {}
+      VisibilityConfig:
+        SampledRequestsEnabled: true
+        CloudWatchMetricsEnabled: true
+        MetricName: WafAclMetric
+      Rules:
+        - Name: AwsCommonRule
+          Priority: 0
+          OverrideAction:
+            None: {}
+          VisibilityConfig:
+            SampledRequestsEnabled: true
+            CloudWatchMetricsEnabled: true
+            MetricName: AwsCommonRuleMetric
+          Statement:
+            ManagedRuleGroupStatement:
+              VendorName: AWS
+              Name: AWSManagedRulesCommonRuleSet
+        - Name: AwsKnownBadRule
+          Priority: 1
+          OverrideAction:
+            None: {}
+          VisibilityConfig:
+            SampledRequestsEnabled: true
+            CloudWatchMetricsEnabled: true
+            MetricName: AwsKnownBadRuleMetric
+          Statement:
+            ManagedRuleGroupStatement:
+              VendorName: AWS
+              Name: AWSManagedRulesKnownBadInputsRuleSet
+        - Name: AwsAdminRule
+          Priority: 2
+          OverrideAction:
+            None: {}
+          VisibilityConfig:
+            SampledRequestsEnabled: true
+            CloudWatchMetricsEnabled: true
+            MetricName: AwsAdminRuleMetric
+          Statement:
+            ManagedRuleGroupStatement:
+              VendorName: AWS
+              Name: AWSManagedRulesAdminProtectionRuleSet
+
+  WafAclAssociation:
+    Type: 'AWS::WAFv2::WebACLAssociation'
+    Properties:
+      ResourceArn: !Ref AWSEBV2LoadBalancer
```

Comment:
i don't think this is gonna work since `AWSEBV2LoadBalancer` is not in here.  

you might have better luck if you move this template to the [BridgeServer2-infra repo](https://github.com/Sage-Bionetworks/BridgeServer2-infra).  you'll probably want to setup something like [bridgeserver2-https-redirect.yaml](https://github.com/Sage-Bionetworks/BridgeServer2-infra/blob/develop/config/develop/bridgeserver2-https-redirect.yaml)

Reply:
Beanstalk let's us [reference the resources created by beanstalk for the environment](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/customize-containers-format-resources-eb.html) in these templates. since our beanstalk environment is web-server tier with load-balancing, beanstalk will create `AWSEBV2LoadBalancer` for us.

Reply:
ohh cool, wondering if this is something new or whether it has always been there and i just didn't know about it.  anyways, wish i knew about this earlier.

Do you know whether it's also possible to magically reference `AWSEBV2LoadBalancer` like this in a cloudformation template when deployed from BridgeServer2-infra repo?

Reply:
No, I don't think you can directly reference `AWSEBV2LoadBalancer` from another template, I think beanstalk might be magically combining this with a cloudformation template it makes behind the scenes. 

but it is possible to export a stack output and reference that from the stack in BridgeServer2-infra.